api/client/network/create.go

```diff
@@ -55,6 +56,7 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	flags.StringSliceVar(&opts.labels, "label", []string{}, "Set metadata on a network")
 	flags.BoolVar(&opts.internal, "internal", false, "Restrict external access to the network")
 	flags.BoolVar(&opts.ipv6, "ipv6", false, "Enable IPv6 networking")
+	flags.BoolVar(&opts.manuallyAttachable, "manually-attachable", false, "Enable manual container attachment")
```

Comment:
@mrjana since the need for this flag is explicitly to adhere to the swarmkit security model, could we name it appropriately (such as `--worker-accessible / -w` and that makes this network accessible by workers from a mgmt standpoint)